### DISCUSSION TITLE
Huge Mismatch between Browser+ and Civitai site

### DISCUSSION THREAD

**Discussion by FifthDream** [BlafKing/sd-civitai-browser-plus#331] [Q&A]
I've used Browser+ exclusively for a while, checking occasionally for newly-released LORAs. A few days ago i went to the actual Civitai site and noticed the models listed on the site (also filtered by newest) seem to bear no relation at all to what is listed in Browser+. I even search in Browser+ for some of the LORAs i see on the site and they don't appear in the results. Is there (a lot of) lag between the site and what shows up in the browser, or is the search imperfect, or am i just doing something wrong?  

I much prefer the browser because it makes downloading LORAs with included thumbnails and info so much easier than it is on the site, but if it's not showing what's available i may have to move on from it. 

Any info/suggestions?

**Comment by Anonym121** [comment]
Browser + hides early access models by default. It could be that
